capstone-project/src/main/angular-webapp/src/app/authentication/authentication.component.ts

```diff
@@ -0,0 +1,31 @@
+import { Component, OnInit } from '@angular/core';
+import { SocialAuthService } from "angularx-social-login";
+import { GoogleLoginProvider } from "angularx-social-login";
+import { SocialUser } from "angularx-social-login";
+
+@Component({
+  selector: 'app-authentication',
+  templateUrl: './authentication.component.html',
+  styleUrls: ['./authentication.component.css']
+})
+export class AuthenticationComponent implements OnInit {
+
+  constructor(private authService: SocialAuthService) { }
+
+  ngOnInit(): void {
+    this.authService.authState.subscribe((user) => {
+      this.user = user;
+    });
+  }
+
+  user: SocialUser;
```

Comment:
Define class members on top of the class. 

This can be private

Reply:
I moved the declaration, but if user will be private we won't be able access it from the template.